Packages/VA FileMan/Globals/0.85+LANGUAGE.zwr

```diff
@@ -501,6 +501,9 @@ OSEHRA ZGO Export: LANGUAGE
 ^DI(.85,196,0)="KONGO^KG^KON^^^M^L"
 ^DI(.85,197,0)="KONKANI^^KOK^^^M^L"
 ^DI(.85,198,0)="KOREAN^KO^KOR^^^I^L"
+^DI(.85,198,20.2)="S:$G(%DT)'[""I"" %DT=$G(%DT)_""I"" G CONT^%DT"
+^DI(.85,198,"DD")="S Y=$$FMTE^UKOUTL(Y,$G(%F))"
```

Comment:
would a long term plan be having these $$FMTE be part of FileMan's i18n?

Reply:
As I mentioned before, I doubt that we can fold these into Fileman itself.